ie-solver/skelfac.cpp

```diff
@@ -3,39 +3,37 @@
 namespace ie_solver{
 
 void Skelfac::GetXMatrices(ie_Mat& K, ie_Mat& Z, ie_Mat& Xrr, 
-	std::vector<int>& r, std::vector<int>& s, std::vector<int>& n) {
+	std::vector<unsigned int>& r, std::vector<unsigned int>& s, 
+	std::vector<unsigned int>& n) {
     
-
-	int rs = r.size();
-	int ss = s.size();
-	int ns = n.size();
+	unsigned int rs = r.size();
+	unsigned int ss = s.size();
+	unsigned int ns = n.size();
     ie_Mat  Xrs(rs, ss), Xsr(ss, rs),
```

Comment:
Is this preallocation even necessary?

Reply:
It's not redundant, if we didn't call the sizing constructor, sizing would happen in assignment, so who cares.